api_emulator/redfish/ComputerSystem_api.py

```diff
@@ -127,19 +128,21 @@ def patch(self, ident):
     def delete(self, ident):
         logging.info('ComputerSystemAPI DELETE called')
         try:
-            resp = 404
             if ident in members:
-                if members[ident]['SystemType'] == 'Composed':
-                    # Delete a composed system
-                    resp = DeleteComposedSystem(ident)
-                    resp = 200
-                else:
-                    # Delete a physical system
-                    del(members[ident])
-                    resp = 200
+                #if members[ident]['SystemType'] == 'Composed':
```

Comment:
Why is this if block remove and the DeleteComposedSystem() not called?  That routine cleans up ResourceBlocks.

Reply:
Good catch! This is another issue that I have found looking at the code. `Composed` is set in `CreateComposedSystem` [here](https://github.com/DMTF/Redfish-Interface-Emulator/blob/master/api_emulator/redfish/ComputerSystem_api.py#L314) but the call is currently commented as part of the `post` [here](https://github.com/DMTF/Redfish-Interface-Emulator/blob/master/api_emulator/redfish/ComputerSystem_api.py#L210). In my opinion, restoring `CreateComposedSystem` and `DeleteComposedSystem` is a large enough change to have its own pull request.

Reply:
Apologies for the delay. Yes we can leave the code block and tackle the problem in a different issue. I will commit an updated version by the end of the day.